src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -60,7 +60,7 @@ internal static byte[] SslGetAlpnSelected(SafeSslHandle ssl)
             int len;
             SslGetAlpnSelected(ssl, out protocol, out len);
 
-            if (len < 1)
+            if (len == 0)
```

Comment:
What about the native change to go along with this?

Reply:
Your too quick, I was using Github to transfer files to my unix box to make the change :P

Reply:
Heh, ok.